doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -948,15 +948,15 @@ maintainance of these query patterns is implemented inside an operator scope, ca
 hierarchical scheme for either reduction or top-k computation.
 
 <!-- RELATION_SPEC mz_internal.mz_expected_group_size_advice -->
-| Field           | Type        | Meaning                                                                                                             |
-|-----------------|-------------|---------------------------------------------------------------------------------------------------------------------|
-| `dataflow_id`   | [`uint8`]   | The ID of the [dataflow]. Corresponds to [`mz_dataflows.id`](#mz_dataflows).                                        |
-| `dataflow_name` | [`text`]    | The internal name of the dataflow hosting the min/max reduction or top-k.                                           |
-| `region_id`     | [`uint8`] | The ID of the root operator scope. Corresponds to [`mz_dataflow_operators.id`](#mz_dataflow_operators).               |
-| `region_name`   | [`text`]    | The internal name of the root operator scope for the min/max reduction or top-k.                                    |
-| `levels`        | [`bigint`] | The number of levels in the hierarchical scheme implemented by the region.                                           |
-| `to_cut`        | [`bigint`] | The number of levels that can be eliminated (cut) from the region's hierarchy.                                       |
-| `hint`          | [`double precision`] | The hint value for `EXPECTED GROUP SIZE` that will eliminate `to_cut` levels from the regions' hierarchy.  |
+| Field           | Type                 | Meaning                                                                                                   |
+|-----------------|----------------------|-----------------------------------------------------------------------------------------------------------|
+| `dataflow_id`   | [`uint8`]            | The ID of the [dataflow]. Corresponds to [`mz_dataflows.id`](#mz_dataflows).                              |
+| `dataflow_name` | [`text`]             | The internal name of the dataflow hosting the min/max reduction or top-k.                                 |
```

Comment:
> reduction

"aggregation" might be a more standard term in SQL? (Maybe @morsapaes will know this.)